addons/binding/org.openhab.binding.freeathome/src/main/java/org/openhab/binding/freeathome/handler/FreeAtHomeBridgeHandler.java

```diff
@@ -393,14 +397,14 @@ private void connectGateway() {
         // m_XmppClient.getManager(PresenceManager.class).requestSubscription(j, "");
         // m_XmppClient.getManager(PresenceManager.class).approveSubscription(j);
 
-        // Presence presence = new Presence(j);
+         Presence presence = new Presence();
```

Comment:
Here an default presence is generated and sent at line 407. 
What happens if I sent a default presence or subscribe @ a specific user such as mrha?